test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -9,8 +9,10 @@
 #include <netinet/in.h>
 #include <arpa/inet.h>
 #include <sys/time.h>
+#include <unistd.h>
```

Comment:
can remove?

Reply:
The ```close``` API function is declared in this header file